packages/fxa-payments-server/src/lib/amplitude.ts

```diff
@@ -70,18 +71,32 @@ export function addGlobalEventProperties(props: GlobalEventProperties) {
 
 export function subscribeToReduxStore(store: Store) {
   let unsubscribe: ReturnType<typeof store.subscribe>;
-  const uidObs = () => {
+  const obs = () => {
     const profile = selectors.profile(store.getState());
+    const subscriptions = selectors.subscriptions(store.getState());
     if (
       profile?.result &&
       profile.result.uid &&
       profile.result.metricsEnabled !== false
     ) {
       addGlobalEventProperties({ uid: profile.result.uid });
-      unsubscribe?.();
     }
+
+    if (
+      subscriptions?.result &&
+      subscriptions.result.length &&
+      profile.result?.metricsEnabled !== false
+    ) {
+      addGlobalEventProperties({
+        subscribed_plan_ids: subscriptions.result
+          .map((sub) => sub.subscriptionId)
+          .join(','),
+      });
+    }
```

Comment:
Hm this logic works except it's not considering `checkout_type` as we'd discussed in Slack. Upon reflection, I think it's better to go this route and forget about the empty string part. I've updated the Tech Spec accordingly.

Reply:
👍